### DISCUSSION TITLE
More then one Stream on the same Server?

### DISCUSSION THREAD

**Discussion by monotron303** [michaelkamprath/multi-service-rtmp-broadcaster#36] [Q&A]
I am not sure that's that possible...
I have two projects and both projects has other Streamingplattforms and need two different configuration files.
My idea is to start the server with other ports but that's not possible...
Is that possible?

And thank you very much for this solution! This ist perfect! I live it very much and it works very good. A better UI where great, I have try it to controle the docker start/stop over a webpage and configure the json file with a Webform. In my idea, it works. To edit the json file, is not the problem but to start/stop the Docker or check the status... I hope, you are faster as I because I think, you where better as me :D

**Comment by michaelkamprath** [comment]
Honestly, I am not sure what you are asking for. Are you asking if you can run two instance of the `multi-service-rtmp-broadcaster`? If so, the answer is "of course". If you use the same server, you need to make sure the external ports that Docker provides each container is different. If you run on different servers, you don't have to worry about port collision.

**Comment by monotron303** [comment]
Thanks for the answer. Yes I mean on the same server. Can you tell me how I can change the external ports? In my opinion it is nit possible to change the ports in OBS!? Or is it the standardly with: url:port?
